recipes/abyss/meta.yaml

```diff
@@ -10,7 +10,7 @@ build:
 
 source:
   url: https://github.com/bcgsc/abyss/releases/download/{{ version }}/abyss-{{ version }}.tar.gz
-  sha256: 6b42edef0c61bc553936d7262d581291712a0cb09da9dd0a6aff466e9cdef28b
+  sha256: 315bad1b4e9038d6ebf45260e906f3bb3cecd99a0b845ec17996dd8846b4920b
```

Comment:
@jowong4 Why did the SHA-256 of ABySS 2.2.0 change?

Reply:
Ka Ming informed me that I forgot to change the README, so I decided since it was still in the early phase of releasing 2.2.0, I may as well re-release it with the modified README

Reply:
It's better to tag a new release (2.2.1) rather than rerelease. Don't worry about this time, for next time.